db/compaction/compaction_service_test.cc

```diff
@@ -86,7 +90,8 @@ class CompactionServiceTest : public DBTestBase {
     for (int i = 0; i < 20; i++) {
       for (int j = 0; j < 10; j++) {
         int key_id = i * 10 + j;
-        ASSERT_OK(Put(Key(key_id), "value" + ToString(key_id)));
+        ASSERT_OK(
+            Put(Key(key_id), "value" + ROCKSDB_NAMESPACE::ToString(key_id)));
```

Comment:
We don't need to specify the namespace here. Only one needed is in CompactionService derived classes, which has name conflict for ToString().

Reply:
Got you!